src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -342,21 +384,47 @@ public void loadGestures() {
   }
 
   /**
-   * This method will look at all of the .py files in a directory. One by one it
+   * This method will try to launch a python command
+   * with error handling
+   */
+  public String execGesture(String gesture) {
```

Comment:
too big a change for nixie.  stuff like this should be inmoov2 and in ogre.

Reply:
No we need it ! not sure about when is the next release... that is optional , gestures are not using it yet..